src/database/services/quiz-api-service.ts

```diff
@@ -1,7 +1,7 @@
 import type { DocumentClient } from "aws-sdk/clients/dynamodb";
 import type QuestionnaireModel from "../models/questionnaire";
 
-const TABLE_NAME = "questionnaires";
+const TABLE_NAME = "Questionnaires";
```

Comment:
I'm a little nervous this might cause problems with the existing dynamoDb table but it should be renamed so lets see what happens!